composer.json

```diff
@@ -19,6 +19,7 @@
     },
     "require": {
         "php": ">=5.5.0",
+        "composer/semver": "^3.2",
```

Comment:
I'm not really happy about adding a dependency on composer/semver here, especially since the only reason it seems to be needed is to identify stable vs non-stable releases. I am ambivalent about suggesting the use of regex's instead.

The issue with composer/semver is that ^1 only works with composer/composer ^1, and composer/semver ^3 only works with composer/composer ^2. While this only affects clients that have a dependency on composer/composer or composer/semver (that is, using this dependency does not constrain the version of the Composer cli you can use), this is still kind of a strict limitation to impose on all Robo clients. Could we at least use `composer/semver: ^1 || ^3.2` here?

Reply:
Sure, I broadened the constraint to include semver 1. I understand the hesitancy to add new dependencies, but I think if we want to align with Composer channels, it only makes sense to use the same Composer version parsing.